docs_2.0/manual_EN/3.quick-start/4.nebula-graph-crud.md

```diff
@@ -50,13 +50,14 @@ From the **Status** column of the table in the return message, you can see that
 * Create a graph space:
 
     ```nGQL
-    CREATE SPACE <graph_space_name>(partition_num=<partition number>, replica_factor=<replica number>);
+    CREATE SPACE <graph_space_name> [(partition_num=<partition number>, replica_factor=<replica number>), vid_type=fixed_string(<string_number>))];
     ```
 
     | Property       | Description                                                                                                                                                                                                                           |
     | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
     | partition_num  | Specifies the number of partitions in each replica. The suggested number is the number of hard disks in the cluster times 5. For example, if you have 3 hard disks in the cluster, we recommend that you set 15 partitions.           |
     | replica_factor | Specifies the number of replicas in the Nebula Graph cluster. The suggested number is 3 in a production environment and 1 in a test environment. The replica number must always be an odd number for the need of quorum-based voting. |
+    |vid_type | To insert vertices with string type IDs, specify the fixed string length for the vertices IDs.|
```

Comment:
To insert a vertex with a string-typed VID. Set fixed_string length for the maximum VID length.  (default value is 8).